### ISSUE TITLE
Narration button doesn't work on some slideshows.

### ISSUE THREAD

**Issue by bedmison** [OpenDSA/OpenDSA#731]
The slideshows are supposed to provide a text-to-speech narration. This feature seems to be broken. 

Look at the slideshows in /RST/en/PL/Grammers2.rst for an example. 

It is unclear if the soundSettings object in /lib/JSAV.js:2199 is correctly built

**Comment by cashaffer** [comment]
Doubtful. We have https://opendsa.readthedocs.io/en/latest/index.html, and there is probably a place that it could go in there to advise people to use umsg for their slides main points.
